packages/app_ui/lib/src/dialogs/bottom_animated_dialog.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flutter/material.dart';
+
+class BottomAnimatedDialog {
+  static void show(BuildContext context, {required Widget child}) {
+    showGeneralDialog(
+      barrierLabel: 'BottomAnimatedDialog',
+      barrierDismissible: true,
+      barrierColor: Colors.black.withOpacity(0.5),
+      transitionDuration: const Duration(milliseconds: 500),
+      context: context,
+      pageBuilder: (context, anim1, anim2) {
+        return Align(
+          alignment: Alignment.bottomCenter,
+          child: Container(
+            height: 300,
+            width: double.infinity,
+            margin: EdgeInsets.only(bottom: 20, left: 12, right: 12),
+            child: child,
+            decoration: BoxDecoration(
+              color: Colors.white,
```

Comment:
everywhere where you put color, let's take the colors from AppColors.
So in this case:
`color: AppColors.white`
for instance